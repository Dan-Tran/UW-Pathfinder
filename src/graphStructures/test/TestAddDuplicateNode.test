# This will test adding nodes to the graph.

# Create the graph.
CreateGraph graphtest

# Make sure the graph has no nodes and no edges.
ListNodes graphtest

# Add one node
AddNode graphtest n1

# Add a duplicate node
AddNode graphtest n1

# Make sure there are no duplicates
ListNodes graphtest

# Add multiple nodes
AddNode graphtest n2
AddNode graphtest n3
AddNode graphtest n4

# Add duplicate nodes
AddNode graphtest n2
AddNode graphtest n1

# Make sure there are no duplicates
ListNodes graphtest

# Add a node
AddNode graphtest n5

# Add dupliate nodes
AddNode graphtest n1
AddNode graphtest n3
AddNode graphtest n5

# Make sure there are no duplicates
ListNodes graphtest