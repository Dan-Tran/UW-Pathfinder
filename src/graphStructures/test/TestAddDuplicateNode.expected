# This will test adding nodes to the graph.

# Create the graph.
created graph graphtest

# Make sure the graph has no nodes and no edges.
graphtest contains:

# Add one node
added node n1 to graphtest

# Add a duplicate node
added node n1 to graphtest

# Make sure there are no duplicates
graphtest contains: n1

# Add multiple nodes
added node n2 to graphtest
added node n3 to graphtest
added node n4 to graphtest

# Add duplicate nodes
added node n2 to graphtest
added node n1 to graphtest

# Make sure there are no duplicates
graphtest contains: n1 n2 n3 n4

# Add a node
added node n5 to graphtest

# Add dupliate nodes
added node n1 to graphtest
added node n3 to graphtest
added node n5 to graphtest

# Make sure there are no duplicates
graphtest contains: n1 n2 n3 n4 n5